app/scripts/modules/amazon/src/pipeline/stages/rollbackCluster/awsRollbackClusterStage.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.amazon.pipeline.stage.rollbackClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'rollbackCluster',
+      cloudProvider: 'aws',
+      templateUrl: require('./rollbackClusterStage.html'),
+      validators: [
+        { type: 'requiredField', fieldName: 'cluster' },
+        { type: 'requiredField', fieldName: 'regions' },
+        { type: 'requiredField', fieldName: 'credentials', fieldLabel: 'account'},
+      ],
+    });
+  }).controller('awsRollbackClusterStageCtrl', function($scope, accountService) {
+    var ctrl = this;
+
+    let stage = $scope.stage;
+
+    $scope.state = {
+      accounts: false,
+      regionsLoaded: false
+    };
+
+    accountService.listAccounts('aws').then(function (accounts) {
+      $scope.accounts = accounts;
+      $scope.state.accounts = true;
+    });
+
+    ctrl.reset = () => {
+      ctrl.accountUpdated();
+      ctrl.resetSelectedCluster();
+    };
+
+    stage.regions = stage.regions || [];
+    stage.cloudProvider = 'aws';
+    stage.targetHealthyRollbackPercentage = stage.targetHealthyRollbackPercentage || 100;
+
+    if (stage.isNew && $scope.application.attributes.platformHealthOnlyShowOverride && $scope.application.attributes.platformHealthOnly) {
+      stage.interestingHealthProviderNames = ['Amazon'];
+    }
+
+    if (!stage.credentials && $scope.application.defaultCredentials.aws) {
+      stage.credentials = $scope.application.defaultCredentials.aws;
+    }
+    if (!stage.regions.length && $scope.application.defaultRegions.aws) {
+      stage.regions.push($scope.application.defaultRegions.aws);
+    }
+
+    ctrl.pluralize = function(str, val) {
```

Comment:
doesn't look like this is being used anywhere

Reply:
👍